gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -530,6 +529,44 @@ private void pollAndAdvanceDag()
       }
     }
 
+    private ExecutionStatus getJobExcecutionStatus(boolean slaKilled, JobStatus jobStatus) {
+      if (slaKilled) {
+        return CANCELLED;
+      } else {
+        if (jobStatus == null) {
+          return PENDING;
+        } else {
+          return valueOf(jobStatus.getEventName());
+        }
+      }
+    }
+
+    /**
+     * Check if the SLA is configured for this job. If it is, tries to cancel the job if SLA is reached.
```

Comment:
Make sure that the comment reflects the fact that we are dealing with flow level SLA and not the job SLA.